raiden/message_handler.py

```diff
@@ -141,10 +141,10 @@ def handle_message_lockexpired(raiden: RaidenService, message: LockExpired) -> N
 
     @staticmethod
     def handle_message_refundtransfer(raiden: RaidenService, message: RefundTransfer) -> None:
-        token_network_address = message.token_network_address
-        from_transfer = lockedtransfersigned_from_message(message)
         chain_state = views.state_from_raiden(raiden)
+        from_transfer = lockedtransfersigned_from_message(message)
```

Comment:
Is there any reason to move these lines?

Reply:
Just to make the code more raiden like. So the code is more "human readable"/like to compare with raiden.